src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3522,7 +3556,6 @@ static char* reserve_large_pages_aligned(size_t size, size_t alignment, bool exe
 char* os::pd_reserve_memory_special(size_t bytes, size_t alignment, size_t page_size, char* addr,
                                     bool exec) {
   assert(UseLargePages, "only for large pages");
-  assert(page_size == os::large_page_size(), "Currently only support one large page size on Windows");
```

Comment:
Would it be better to assert that page_size is at least GetLargePageMinimum() instead of removing this assert?